src/pages/ThorChainLP/components/AddLiquitity/AddLiquidityInput.tsx

```diff
@@ -823,23 +825,19 @@ export const AddLiquidityInput: React.FC<AddLiquidityInputProps> = ({
         outputExponent: THOR_PRECISION,
       }).toFixed()
 
-      setIsSlippageLoading(true)
-
       const estimate = await estimateAddThorchainLiquidityPosition({
         runeAmountCryptoThorPrecision,
         assetAmountCryptoThorPrecision,
         assetId: poolAsset.assetId,
       })
 
+      const slippageRune = bnOrZero(estimate.slipPercent)
+        .div(100)
+        .times(virtualRuneFiatLiquidityAmount ?? 0)
+        .times(2)
+        .toFixed()
```

Comment:
not intro'd here but can u add a comment to explain the math? I can see myself squinting at this during a prod issue

Reply:
gm https://github.com/shapeshift/web/pull/6282/commits/1941940deae3937702914318c213b815e43904d0